README.md

```diff
@@ -179,10 +179,100 @@ If you're calling a **GET** on `/entities` , you're asking for a list of entitie
 - **GET**  `/media/:media_id/update_comment?text=comment%20new%20text`
 > Deleting a comment on a media item:
 
-- **GET** `/media/:media_id/delete_comment?id=comment1`
+- **GET** `/media/:media_id/delete_comment?id=:id`
 
 
 ##### Reasoning
-Using the right HTTP Verb on each resource will make clearer what the resource is made for and what we should expect it to do, receive and return. 
+Using the right HTTP Verb on each endpoint will make clearer what the endpoint is made for and what we should expect it to do, receive and return. 
+
+---
+
+#### Use Nouns for endpoints names (URLs)
+> Your endpoints names should be always nouns, not verbs.
+
+##### Examples
+
+###### Good
+
+> Getting a user
+
+- **GET** `/users/:id`
+  The endpoint retrieves the user with the id you pass along in the usl.
+
+> Creating a user:
+
+- **POST** `/users`
+  This creates a user with the info you pass along in the request body.
+
+> Deleting a post
+
+- **DELETE** `/users/:id`
+  This Deletes the user with the id you pass along in the url
+
+###### Bad
+
+> Getting a post
+
+- **GET** `/get/posts?id=:id`
+  This is wrong because you already established that it is a GET. Using /get on the endpoint name is redundant
+
+> Creating a post
+
+- **POST** `/create/post`
+  Same as above, you already estabished that you're going to create something with the **POST** so, again, the create is redundant
+
+> Deleting a post
+
+- **POST** `/delete_post/:id`
+  In thi case not only you're being redundant by putting the verb delete on the endpoint name, but you're doing a wrong redundancy because it is actually a **POST** request and they're not for deleting objects
+##### Reasoning
+
+Your endpoind should reflect what they do based on the HTTP Verb plus the endpoing name (noun). So it make sense that doing a **GET** to `/entities` retrieves a list of entities. In exchange, doing a **GET** to `/get/entity` looks redundant and is not self descriptive. One more example: **POST** to `/update_entity/:id` that, semantically speaking, means you're Creating an `update_entity` object with the id on the url IS WRONG. This should be achived by **PUT** or **PATCH** `/entities/:id`
```

Comment:
So it make -> Therefore, it makes
In exchange -> On the contrary
and is not -> and it's not
One more example: … -> Let's try with another example: **POST** `/update_entity/:id`. That, semantically speaking, should be interpreted as creating/adding an entity to the `update_entity/:id` set.  If your intention was to update the entity with id `:id`, you should've used **PUT** or **PATCH** `/entities/:id` instead.